File: gems/decomplex/examples/swift/semantic-alias.swift
func frame(node: Node) -> Bool { return node.provenance == FRAME }
func is_frame(node: Node) -> Bool { return provenance == FRAME }
func heap(node: Node) -> Bool { return node.provenance == HEAP }
func somewhere(node: Node) -> Int { if node.provenance == FRAME { return 1 }; return 0 }

File: gems/decomplex/examples/go/semantic-alias.go
package main
func frame(node Node) bool { return node.provenance == FRAME }
func is_frame(node Node) bool { return provenance == FRAME }
func heap(node Node) bool { return node.provenance == HEAP }
func somewhere(node Node) int { if node.provenance == FRAME { return 1 }; return 0 }
